src/EnergyPlus/Autosizing/HeatingWaterDesAirInletTempSizing.cc

```diff
@@ -160,24 +160,22 @@ EnergyPlus::AutoSizingResultType HeatingWaterDesAirInletTempSizer::size(EnergyPl
     } else if (this->curSysNum > 0) {
         if (!this->wasAutoSized && !this->sizingDesRunThisAirSys) {
             if (this->printWarningFlag && this->originalValue > 0.0) {
-                this->reportSizerOutput(
+                HeatingWaterDesAirInletTempSizer::reportSizerOutput(
```

Comment:
I don't see the difference here. The report functions are a member of the base class.

Reply:
Good question.  `reportSizerOutput` is marked as static.  It technically isn't a class method that operates on a single instance of a sizer.  It is a function that just happens to be organized to be within the base sizer class.  When you call static functions, you just call them with the namespace/class name to differentiate that you are calling a static object, not operating on a specific instance of a class.  I can elaborate if you'd like!